src/BookList.tsx

```diff
@@ -36,7 +38,7 @@ async function newBook(dispatch) {
   } else {
     const book = res.payload;
     console.log("new book", book);
-    dispatch({ type: "ADD_BOOK", payload: book });
+    dispatch(librarySlice.actions.addBook(book));
```

Comment:
If you look at the quick start we can make this even easier:
https://redux-toolkit.js.org/tutorials/quick-start

by doing something like: `export const { increment, decrement, incrementByAmount } = librarySlice.actions`

The problem is we have A LOT of actions. I think this is something we can reevaluate and clean up later